src/constant/index.ts

```diff
@@ -530,6 +537,7 @@ export enum WALLET_BRAND_TYPES {
   MPCVault = 'MPCVault',
   Coinbase = 'Coinbase',
   IMKEY = 'IMKEY',
+  NGRAVEZERO = 'NGRAVE',
```

Comment:
```suggestion
  NGRAVEZERO = 'NGRAVE ZERO',
```

Reply:
Don't add extra code to change the `brandName`, just set the brand to what you need.